Packs/ManageEngine/Integrations/ManageEngineEventCollector/ManageEngineEventCollector_description.md

```diff
@@ -0,0 +1,35 @@
+# Endpoint Central Cloud Domains
+
+Endpoint Central cloud is hosted at multiple data centers, and therefore available on different domains. There are 7 different domains for Endpoint central cloud APIs, so you can use the one that is applicable to you.
```

Comment:
Endpoint Central cloud is hosted at multiple data centers, and therefore available on different domains. There are several domains for Endpoint Central Cloud APIs, so you can use the one that is applicable to you.